src/core/listener.c

```diff
@@ -304,6 +304,11 @@ MsQuicListenerStart(
         Listener,
         Listener->Binding,
         CLOG_BYTEARRAY(sizeof(Listener->LocalAddress), &Listener->LocalAddress));
+    QuicTraceEvent(
+        ListenerAlpn,
+        "[list][%p] Accepting ALPNs: %!ALPN!",
+        Listener,
+        CLOG_BYTEARRAY(Listener->AlpnListLength, Listener->AlpnList));
```

Comment:
Can you just append this to the `ListenerStarted` event instead? Appending to existing events is safe to do.

Reply:
It's not going to break the existing logs? It'll change the string table entry, which will make localization upset.

Reply:
We don't localize our strings. So it's fine.